### PR TITLE
Working thread

### PR DESCRIPTION
**Pull request by dmalia1**

### CODE REVIEW

File: src/awscli_login/__main__.py [hunk 1]
```diff
@@ -141,9 +149,51 @@ def wrapper(args: Namespace, session: Session):
...
+            # Use the highest available protocol
+            pickle.dump(worker_argv, f, protocol=-1)
+
+        # Create an env for the worker to let it know what to do
+        worker_env = {}
+
+        worker_env.update(dict(os.environ))
+        # Figure out the path to the current file
+        # worker_target = os.path.abspath(__file__)
+        worker_cmd = ('"' + python_path + '" -m ' +
```
Comment (keithbauer) [inline src/awscli_login/__main__.py]: If this is running on python 3.6 or newer then f-strings might make this more readable. Something like `worker_cmd = f"{python_path} -m awscli_login.windaemon {worker_argpath}"`

File: src/awscli_login/windaemon.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+def daemonize_for_windows(profile, role, expires):
+    pidfile = profile.pidfile
+    with Daemonizer() as (is_setup, daemonizer):
+        is_parent, profile, role, expires = daemonizer(
+            pidfile, profile, role, expires
+        )
+        if not is_parent:
+            logger = configFileLogger(profile.logfile, logging.INFO)
+            logger.info('Startig refresh process for role %s' % role[1])
```
Comment (keithbauer) [inline src/awscli_login/windaemon.py]: Startig => Starting

**Review by keithbauer — COMMENTED** [review]
Just a spelling change and a minor formatting suggestion but other than that I think this looks good. Not sure if there was a conflict for renaming typing to awscli_typing, but as long as it works.

File: src/awscli_login/__main__.py (hunk 1 above)
Comment (dmalia1) [reply src/awscli_login/__main__.py]: I'm not sure everyone using this is using > 3.6 python.

Comment (dmalia1) [reply src/awscli_login/__main__.py]: I went ahead and changed it.

**Pull request merged**